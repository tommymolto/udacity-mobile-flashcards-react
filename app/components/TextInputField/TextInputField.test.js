/* global describe, it, expect */

import React from 'react';
import renderer from 'react-test-renderer';
import TextInputField from './TextInputField';

describe('TextInputField', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<TextInputField
        fieldLabel="label"
        placeholder="placeholder"
        maxLength={40}
        onChangeText={() => { }} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
