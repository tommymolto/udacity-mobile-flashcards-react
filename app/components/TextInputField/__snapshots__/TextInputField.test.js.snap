// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInputField renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "paddingHorizontal": 15,
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 16,
          "paddingTop": 10,
        },
        undefined,
      ]
    }
  >
    label
  </Text>
  <TextInput
    allowFontScaling={true}
    blurOnSubmit={true}
    keyboardType="default"
    maxLength={40}
    onChangeText={[Function]}
    placeholder="placeholder"
    style={
      Object {
        "height": 40,
      }
    }
  />
</View>
`;
